src/components/Home.jsx

```diff
@@ -1,7 +1,44 @@
 import React from 'react';
+import getToken from '../lib/tokens';
+import { Redirect } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
```

Comment:
These imports can actually be combined into one line.
```jsx
import { Redirect, useHistory } from 'react-router-dom';
```

Reply:
Implemented change!